### PR TITLE
fix: reload on logo click

### PR DESCRIPTION
**Pull request by PriteshKiri**

### CODE REVIEW

File: src/components/HeaderLogo.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```
Comment (codewithvoid) [inline src/components/HeaderLogo.js:1]: is this import required?

File: src/components/HeaderLogo.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+const HeaderLogo = () => {
```
Comment (codewithvoid) [inline src/components/HeaderLogo.js:3]: nit: use export default function to avoid declaration of extra variable

**Comment by codewithvoid** [comment]
Thanks @PriteshKiri for taking this change. Added small comments.

**Comment by PriteshKiri** [comment]
I have made the changes @codewithvoid. Can you please check?

**Comment by codewithvoid** [comment]
> @PriteshKiri looks good. can you revert the additional files. style fixes can be sent separately if needed.

I have submitted the styling changes separately, you can sync your branch now

**Comment by PriteshKiri** [comment]
Done! Branch is now in sync with main

**Comment by codewithvoid** [comment]
Perfect. Thanks for making the changes.

**Pull request merged**